### ISSUE TITLE
A11y : Cannot switch between Viz/Suggestions/LibraryPanels via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46491]
**Steps to reproduce**:

try switching between Visualizations/Suggestions/Library panels using only a keyboard

**Actual Result**:

focus is lost and Search field is focused instead

**Expected Result**

workflow accessible for keyboard-only users

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
